src/NuGet.Clients/NuGet.CommandLine/Commands/InstallCommand.cs

```diff
@@ -224,6 +224,20 @@ private async Task PerformV2RestoreAsync(string packagesConfigFilePath, string i
                     ClientPolicyContext = clientPolicyContext
                 };
 
+                IEnumerable<SourceRepository> sourceRepositories = packageRestoreContext.SourceRepositories;
+                IEnumerable<SourceRepository> enabledSources = (sourceRepositories != null && sourceRepositories.Any()) ? sourceRepositories :
+                    sourceRepositoryProvider.GetRepositories().Where(e => e.PackageSource.IsEnabled);
+
+                foreach (SourceRepository enabledSource in enabledSources)
+                {
+                    PackageSource source = enabledSource.PackageSource;
+                    if (source.IsHttp && !source.IsHttps)
+                    {
+                        projectContext.Log(RestoreLogMessage.CreateWarning(NuGetLogCode.NU1803,
```

Comment:
We don't code the packages.config warnings. 
You don't need to log a restore message.